libraries/AP_RangeFinder/AP_RangeFinder.cpp

```diff
@@ -683,7 +683,51 @@ float RangeFinder::distance_orient(enum Rotation orientation) const
 
 uint16_t RangeFinder::distance_cm_orient(enum Rotation orientation) const
 {
-    return distance_orient(orientation) * 100.0;
+    uint16_t dist_cm;
+    int8_t signal_quality_pct;
+    distance_cm_orient_ex(orientation, dist_cm, signal_quality_pct);
```

Comment:
Perhaps safer / easier to not change this function, but just introduce new functionality in the _ex function.

Reply:
Good idea. But the old function has a (unlikely) bug that an out-of-range float could be crammed into a unint16_t. Actually, I don't know the conversion behavior of the old assignment. Maybe it does an appropriate thing. If that is the case, maybe the new function should not do any range checking either.